### PR TITLE
Fix document omissions.

### PR DESCRIPTION
**Pull request by heihaozi**
Please provide a description for what this PR is for:

**When cleaning up, delete the `fortio` service.**

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/circuit-breaking/index.md [hunk 1]
```diff
@@ -261,5 +261,5 @@ test: yes
 
     {{< text bash >}}
     $ kubectl delete deploy httpbin fortio-deploy
-    $ kubectl delete svc httpbin
```
Comment (zirain) [inline content/zh/docs/tasks/traffic-management/circuit-breaking/index.md:264]: would be better change both the en and zh doc to following:
```
kubectl apply -f @samples/httpbin/sample-client/fortio-deploy.yaml@
kubectl apply -f @samples/httpbin/httpbin.yaml@
```

Comment (heihaozi) [reply content/zh/docs/tasks/traffic-management/circuit-breaking/index.md:264]: OK, this is a good idea.

Comment (zirain) [reply content/zh/docs/tasks/traffic-management/circuit-breaking/index.md:264]: this's a testable doc, please run `make gen` to update the scripts before submit your commit.

**Comment by yanrongshi** [comment]
ok-to-test

**Comment by craigbox** [comment]
Just to check, the original document did

```
    $ kubectl delete deploy httpbin fortio-deploy
    $ kubectl delete svc httpbin fortio
```

What more does the cleanup do if you use the files?

**Comment by zirain** [comment]
actually, right now there're same, you can verify here:
https://github.com/istio/istio/blob/master/samples/httpbin/sample-client/fortio-deploy.yaml
https://github.com/istio/istio/blob/master/samples/httpbin/httpbin.yaml

but, if some add more resources in the future, the old way would be broken.

**Review by ericvn — APPROVED** [review]
LGTM. Thanks

**Pull request merged**